datalad/distribution/create_sibling_github.py

```diff
@@ -100,12 +100,18 @@ class CreateSiblingGithub(Interface):
             constraints=EnsureStr()),
         existing=Parameter(
             args=("--existing",),
-            constraints=EnsureChoice('skip', 'error', 'reconfigure'),
+            constraints=EnsureChoice('skip', 'error', 'reconfigure', 'replace'),
             metavar='MODE',
             doc="""desired behavior when already existing or configured
-            siblings are discovered. 'skip': ignore; 'error': fail immediately;
-            'reconfigure': use the existing repository and reconfigure the
-            local dataset to use it as a sibling""",),
+            siblings are discovered. In this case, a dataset can be skipped
+            ('skip'), the sibling configuration be updated ('reconfigure'),
+            or process interrupts with error ('error'). DANGER ZONE: If 'replace'
+            is used, an existing github repository will be irreversibly removed,
+            re-initialized, and the sibling (re-)configured (thus implies 'reconfigure').
+            `replace` could lead to data loss, so use with care.  To minimize
+            possibility of data loss, in interactive mode DataLad will ask for
+            confirmation, but it would just issue a warning and proceed in
+            non-interactive mode.""",),
```

Comment:
> but it would just issue a warning and proceed in non-interactive mode

This part seems wrong: isn't the behavior (here and for `create-sibling`) to abort rather than issue a warning and proceed?  Perhaps it describes an iteration of the `create-sibling` behavior.

Reply:
THANKS! Fixing up for both create-siblings